client/src/app/shared/components/project-list/project-list.component.ts

```diff
@@ -61,6 +67,7 @@ export class ProjectListComponent implements OnInit {
       [ProjectColumn.short_name]: 'HCA Project ID',
       [ProjectColumn.project_title]: 'Project Title',
       [ProjectColumn.last_updated]: 'Last Updated',
+      [ProjectColumn.wrangling_status]: 'Wrangling Status',
       [ProjectColumn.primary_contributor]: 'Primary Contributor',
       [ProjectColumn.primary_wrangler]: 'Primary Wrangler'
```

Comment:
I've checked that these columns don't show if the user is not a wrangler...
But I can't see the code that hides them, could you point me in the right direction?

Reply:
I've only added them in all-projects component https://github.com/ebi-ait/ingest-ui/pull/94/files#diff-68fc70737bc6a261cf75d759bd0a9ecdef83917b60b1c91c78ebb300bf227d7cR24-R25

Reply:
Yup, sorry i didnt see the errors, thanks for checking! fixing them now!

Reply:
You could specify the columns to include when you use the project-list component.
The additional columns are only added in the all-projects and not in the my-projects component.

Reply:
Found the choice is made here: https://github.com/ebi-ait/ingest-ui/blob/fa031d621c1df39867259a9d011a4d2abec8141a/client/src/app/my-projects/my-projects.component.ts#L43

Adding in context for prosperity.